frontend/src/lib/AdminDetails.svelte

```diff
@@ -0,0 +1,48 @@
+<!-- Copyright 2023 DDS Permissions Manager Authors-->
+<script>
+	import { isAuthenticated } from '../stores/authentication';
+	import { onMount } from 'svelte';
+	import { httpAdapter } from '../appconfig';
+
+	export let groupId;
+	export let adminCategory;
+
+	// Promises
+	let promise;
+
+	// Admins
+	let groupMembership;
+	let admins;
+
+	const fetchAndUpdateAdmin = async () => {
+		let adminEmails = [];
+
+		promise = await httpAdapter.get(`/group_membership?page=0&size=100&group=${groupId}`);
```

Comment:
The admins for a particular item are available in the JSON.  A call to `group_membership` should not be required.  Talk to @montesmoci for the details.

Reply:
Sorry, I should've mentioned before - The backend hosts swagger documentation. For Topics, see http://localhost:8080/swagger-ui#/topic/index_10 . Observe that in the Response section, there is a `admin` key where the value is a list of string.

Reply:
Thanks, I didn't see those `/show` endpoints!